src/Traits/Resizable.php

```diff
@@ -22,6 +22,19 @@ public function thumbnail($type, $attribute = 'image')
         // We take image from posts field
         $image = $this->attributes[$attribute];
 
+        return $this->get_thumbnail($image, $type);
+    }
+
+    /**
+     * Generate thumbnail URL.
+     *
+     * @param $image
+     * @param $type
+     *
+     * @return string
+     */
+    public function get_thumbnail($image, $type)
```

Comment:
I think it would be more clear if we used `camelCase` and went with `getThumbnail ` over `get_thumbnail`. Thoughts?

Reply:
Thanks @marktopper I will change that. I will open new PR too, because my mistake that create new branch to main repo instead of my own fork